go/client/cmd_secretkey.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	rpc "github.com/keybase/go-framed-msgpack-rpc"
+	"golang.org/x/net/context"
+)
+
+type CmdSecretKey struct {
+	libkb.Contextified
+	keytype string
+}
+
+func (c *CmdSecretKey) ParseArgv(ctx *cli.Context) error {
+	if len(ctx.Args()) != 1 {
```

Comment:
I prefer not to use flags for arguments that are mandatory. When I see

    keybase secretkey --signing

I tend to assume that `keybase secretkey` is valid by itself, which it isn't. An alternative might be to define two commands, maybe `signing-key` and `encryption-key` or something. Thoughts?